include/inet.h

```diff
@@ -139,8 +139,17 @@ int pr_inet_reverse_dns(pool *, int);
 int pr_inet_getservport(pool *, const char *, const char *);
 pr_netaddr_t *pr_inet_getaddr(pool *, const char *, array_header **);
 conn_t *pr_inet_copy_conn(pool *, conn_t *);
-conn_t *pr_inet_create_conn(pool *, int, const pr_netaddr_t *, int, int);
-conn_t *pr_inet_create_conn_portrange(pool *, const pr_netaddr_t *, int, int);
+
+conn_t *pr_inet_create_conn(pool *p, int fd, const pr_netaddr_t *bind_addr,
+  int port, int retry_bind);
+conn_t *pr_inet_create_conn2(pool *p, int fd, const pr_netaddr_t *bind_addr,
+  int port, int flags);
+#define PR_INET_CREATE_CONN_FL_RETRY_BIND	0x001
+#define PR_INET_CREATE_CONN_FL_LOG_ERRORS	0x002
```

Comment:
With these new flags, particularly the `LOG_ERRORS` flag, callers can explicitly say whether they want the verbose (confusing?) messages logged, or not.